### PR TITLE
Add Device protected route to disable skill edit in local mode #2744

### LINKED ISSUE #2744: Disable public skill editing in the local build
**Expected Behaviour**
 When the web client is running on the smart speaker, the editing of the public skills should be disabled.
I see two options for this - 
* The localbuild has the edit skills option disabled for public skills
* Disable it on the device via the installer

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2744, #2767

Changes:

 - [x] Added Device protected routes, ie. when on the local device, edit of skill will give message shown in the screenshot. Same for when the user tries to edit using compare id in skill version

- [x] Fix Delete Skill Alignment in SkillCreator

Demo Link: https://pr-2776-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 

Screenshot 2019-08-03 at 1 14 45 AM

Screenshot 2019-08-03 at 1 17 06 AM

### CODE REVIEW

**Comment by fragm3** [comment]
@alok760, please review

File: src/components/shared/DeviceProtectedRoute.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    line-height: 1;
+  }
+`;
+
+const RenderAccessFromSUSI = () => (
+  <Container>
+    <a href="https://susi.ai">
+      <SusiLogo />
+    </a>
+    <h2>Access to this page is restricted from local device,</h2>
```
Comment (akshatnitd) [inline src/components/shared/DeviceProtectedRoute.js]:
```
<h2>
Access to this page is restricted from local device
<br />
Try accessing from public SUSI.AI site
</h2>
```

Comment (akshatnitd) [reply src/components/shared/DeviceProtectedRoute.js]: Use this.

File: src/components/shared/DeviceProtectedRoute.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    <h2>try accessing from live website</h2>
+    <a href="https://susi.ai">
+      <Button variant="contained" color="primary">
```
Comment (akshatnitd) [inline src/components/shared/DeviceProtectedRoute.js]: Should we pass routeProps to this component and redirect to exact page on susi.ai?

Comment (fragm3) [reply src/components/shared/DeviceProtectedRoute.js]: Yes, that would be great, on it

Comment (akshatnitd) [reply src/components/shared/DeviceProtectedRoute.js]: @fragm3 Also, use `https://susi.ai` from urls.js

**Comment by fragm3** [comment]
@akshatnitd made the changes:
Screenshot 2019-08-03 at 1 49 00 AM

File: src/components/shared/DeviceProtectedRoute.js (hunk 1 above)
Comment (akshatnitd) [reply src/components/shared/DeviceProtectedRoute.js]: @fragm3 This change not made

File: src/components/shared/DeviceProtectedRoute.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+const RenderAccessFromSUSI = ({ pathname }) => {
+  return (
+    <Container>
+      <a href={urls.HOME_URL}>
+        <SusiLogo />
+      </a>
+      <h2>
+        Access to this page is restricted from local device,
+        <br />
+        try accessing from live website
```
Comment (akshatnitd) [inline src/components/shared/DeviceProtectedRoute.js]: Make channge here as suggested in previous review.

Comment (fragm3) [reply src/components/shared/DeviceProtectedRoute.js]: Oh, sorry

**Comment by fragm3** [comment]
@akshatnitd made the changes

**Pull request merged**